src/ImageFiltering.jl

```diff
@@ -18,7 +18,9 @@ export Kernel, KernelFactors,
     imfilter, imfilter!,
     mapwindow, mapwindow!,
     imgradients, padarray, centered, kernelfactors, reflect,
-    freqkernel, spacekernel
+    freqkernel, spacekernel,
+    findlocalminima, findlocalmaxima,
+    blob_LoG, BlobLoG
```

Comment:
Do we really need to export the struct `BlobLoG`?

Reply:
Together with my kwarg method, it makes the output cut/pasteable ("round-trippable"), but maybe that's not important.